doc/ref_model/chapters/chapter03.md

```diff
@@ -98,17 +98,25 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 
 <a name="3.2.3"></a>
 ### 3.2.3 Storage
-A block device of a certain size for persisting information which can be created and dynamically attached to/detached from a virtual compute. A storage device resides in a tenant context and exists independently from any compute host.
 
-_**Example**: an OpenStack cinder volume._
+A workload can request different type of data availibility : persistent or ephemeral storage.
+Persistent storage outlives the compute instance whereas ephemeral storage is linked to compute instance lifecycle.
 
-| Attribute      | Description                                                              |
-|----------------|--------------------------------------------------------------------------|
-| `name`         | name of storage resources                                                |
-| `size`         | size of disc in GB                                                       |
-| `attachments`  | list of compute hosts to which the device is currently attached          |
-| `acceleration` | key/value pairs for selection of the appropriate acceleration technology |
-| `metadata`     | key/value pairs for selection of the appropriate redundancy domain       |
+Performance requirements are multiple depending on the workload to be hosted:low latency, IOPS oriented and capacity oriented storage. 
```

Comment:
```suggestion
There are multiple storage performance requirements such as latency, IOPS and capacity. For example, a workload may require one of its storage device to provide low latency, high IOPS and very large/hige storage capacity (terabytes of data).
```

Reply:
May I suggest that capacity be separated from performance -- see storage comment below

Reply:
@pgoyal01 Thanks, suggestion adopted
I will modify the typo hige->huge